packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -123,9 +83,10 @@ export const ConnectedServices = () => {
               {...{
                 key: client.clientId,
                 name: client.name,
+                deviceType: client.deviceType,
                 location: client.location,
                 lastAccessTimeFormatted: client.lastAccessTimeFormatted,
-                canSignOut: false,
+                canSignOut: true,
```

Comment:
Can this ever be false? If it's always true does it need to be a prop?

Also, should I be able to sign out of the session I am currently active on from here?

Reply:
This is addressed in a separate issue(already linked in reply to jared's comment), the jira task that this pr is taking care of asks for a dummy button

Reply:
https://github.com/mozilla/fxa/pull/6613#discussion_r501314042